<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>631</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Canal Resort</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Room Type</string>
      </property>
      <widget class="QRadioButton" name="queenRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2 Queen Bed</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="kingRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1 King Bed</string>
       </property>
      </widget>
     </widget>
     <widget class="QSpinBox" name="nightsSpinBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Room View Type</string>
      </property>
      <widget class="QRadioButton" name="standardRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Standard</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="atriumRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Atrium</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="customerName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>Mm/dd/yyyy</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>First Day of Stay</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="childrenBox">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>130</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="adultBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>130</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adults 18+</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>100</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Children</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="parkingCheckBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>190</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Parking</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Estimated total: </string>
      </property>
     </widget>
     <widget class="QLabel" name="priceLabel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>270</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nights</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPushButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="backPushButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>260</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>-10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cost</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="creditCardLineEdit">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Room:</string>
      </property>
     </widget>
     <widget class="QLabel" name="roomCostLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parking: </string>
      </property>
     </widget>
     <widget class="QLabel" name="parkingLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tax:</string>
      </property>
     </widget>
     <widget class="QLabel" name="taxLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Resort:</string>
      </property>
     </widget>
     <widget class="QLabel" name="resortLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalCostLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QRadioButton" name="visaRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>VISA</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="masterCardRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>MasterCard</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="discoverRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Discover</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="americaexpressRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>American Express</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="payPushButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Pay</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="experationDateLineEdit">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>99/99</string>
      </property>
      <property name="text">
       <string>01/20</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reservation Confirmation</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
